src/os/os_chmod_test.go

```diff
@@ -0,0 +1,27 @@
+// +build !baremetal,!js,!wasi
+
+// TODO: Move this into os_test.go (as upstream has it) when wasi supports chmod
```

Comment:
Should this keep the license header from `os_test.go`?

Reply:
I keep forgetting that...